testnets/xiontestnet/chain.json

```diff
@@ -105,59 +97,129 @@
       },
       {
         "name": "v0.3.7",
-        "tag": "v0.3.7",
+        "tag": "v4.0.0",
+        "proposal": 18,
         "height": 6887750,
-        "recommended_version": "v0.3.7",
+        "go_version": "1.21",
         "cosmos_sdk_version": "0.47.10",
         "cosmwasm_enabled": true,
         "cosmwasm_version": "0.45.0",
         "ibc_go_version": "v7.3.1",
         "consensus": {
           "type": "cometbft",
           "version": "0.37.4"
+        },
+        "binaries": {
+          "linux/amd64": "https://github.com/burnt-labs/xion/releases/download/v0.3.7/xiond-v0.3.7-linux-amd64:checksum=sha256:3d7626e971bf2ece28d4835d45ee6bd104a54ba6d652cd4982fc00ff8a6590de"
         }
       },
       {
         "name": "v0.3.8",
-        "tag": "v0.3.8",
+        "tag": "v5.0.0",
+        "proposal": 19,
         "height": 7130100,
-        "recommended_version": "v0.3.8",
+        "go_version": "1.21",
         "cosmos_sdk_version": "0.47.10",
         "cosmwasm_enabled": true,
         "cosmwasm_version": "0.45.0",
         "ibc_go_version": "v7.3.1",
         "consensus": {
           "type": "cometbft",
           "version": "0.37.4"
+        },
+        "binaries": {
+          "linux/amd64": "https://github.com/burnt-labs/xion/releases/download/v0.3.8/xiond-v0.3.8-linux-amd64:checksum=sha256:ef5112a6bb01e9dd297abf27ab423582004d3261f3300cef419937555bf8e281"
         }
       },
       {
         "name": "v0.3.9",
-        "tag": "v0.3.9",
+        "tag": "v6.0.0",
```

Comment:
I'm slightly suspicious about the discrepancy between name (e.g., v0.3.9) and tag (e.g., v.6.0.0) for many of the earlier versions. (although the later versions seem to have them match pretty closely)